daemon/config_solaris.go

```diff
@@ -17,13 +28,23 @@ type Config struct {
 	CommonConfig
 
 	// Fields below here are platform specific.
-	ExecRoot string `json:"exec-root,omitempty"`
+	ExecRoot       string                   `json:"exec-root,omitempty"`
```

Comment:
Hum this is more a *note for later* than anything, but these are common with othe `unixes`, we might want to group them :angel: